### PR TITLE
LinusLausberg/Exercise02

### PR DESCRIPTION
**Pull request by LinusL2005**

### CODE REVIEW

File: Exercises/exercises_02.py [hunk 1]
```diff
@@ -40,3 +76,42 @@
...
+    print('Dies ist die Liste alles einstelligen Charakter:' , one_charakter_list)
+    print('Es sind' , len(one_charakter_list) , 'Elemente')
+
+           
+
+test1 = ['Tafel' , '1' , '0' , '15' , 'Baum' , '2.0' , 'g' , 'f' , 'w']
+test2 = ['Hannah' , 'Fabian' , 'Philipp' , 'Linus', '1' , '5' , 'j' , '17']
+test3 = ['1', '2.8' , '/' , '' , 'k/m' , 'Sticker' , 'q' , 'k']
+
+print()
```
Comment (PixelWelt) [inline Exercises/exercises_02.py:103]: Use \n at the end of a Print Statement for a new Line or the end="\n \n" Argument

File: Exercises/exercises_02.py [hunk 2]
```diff
@@ -40,3 +76,42 @@
...
+
+print('Test der Testreihe 1. Es sollen folgende Zahlen ausgebenwerden: 1, 0, 15, 2.0. Insgesamt 4 Zahlen')
```
Comment (PixelWelt) [inline Exercises/exercises_02.py:105]: Take a Look into pytest for writing Test cases, that can bei automated.

File: Exercises/exercises_02.py [hunk 3]
```diff
@@ -27,8 +27,44 @@
...
+def multiplication(start_m):
+    solution = working_list[working_list[start_m+1]] * working_list[working_list[start_m+2]]
+    working_list[working_list[start_m+3]] = solution
+
+
+#mainprogram
+
 commands = [1, 12, 2, 3, 1, 1, 2, 3, 1, 3, 4, 3, 1, 5, 0, 3, 2, 1, 9, 19, 1, 5, 19, 23, 1, 6, 23, 27, 1, 27, 10, 31, 1, 31, 5, 35, 2, 10, 35, 39, 1, 9, 39, 43, 1, 43, 5, 47, 1, 47, 6, 51, 2, 51, 6, 55, 1, 13, 55, 59, 2, 6, 59, 63, 1, 63, 5, 67, 2, 10, 67, 71, 1, 9, 71, 75, 1, 75, 13, 79, 1, 10, 79, 83, 2, 83, 13, 87, 1, 87, 6, 91, 1, 5, 91, 95, 2, 95, 9, 99, 1, 5, 99, 103, 1, 103, 6, 107, 2, 107, 13, 111, 1, 111, 10, 115, 2, 10, 115, 119, 1, 9, 119, 123, 1, 123, 9, 127, 1, 13, 127, 131, 2, 10, 131, 135, 1, 135, 5, 139, 1, 2, 139, 143, 1, 143, 5, 0, 99, 2, 0, 14, 0]
 
+working_list = commands
```
Comment (PixelWelt) [inline Exercises/exercises_02.py:60]: I would Put this logic into a function

**Review by PixelWelt — APPROVED** [review]
The Code works as needed, but can bei improved as mentioned in the comments

File: Exercises/exercises_02.py [hunk 4]
```diff
@@ -40,3 +76,42 @@
 # Think of some good inputs to test this functionality, write down at least three
 # examples and verify that the output for these examples is correct.
 
+
+def sorting(input_list):
```
Comment (PixelWelt) [inline Exercises/exercises_02.py:80]: we should use args as input

**Review by PixelWelt — CHANGES_REQUESTED** [review]

File: Exercises/exercises_02.py (hunk 2 above)
Comment (LinusL2005) [reply Exercises/exercises_02.py:105]: ja das hab ich auch überlegt, weil ich das letzte Woche schon ziemlich cool fand. Steht auf der to-Do Liste

File: Exercises/LinusLausberg/exercies_02 [hunk 5]
```diff
@@ -40,3 +76,36 @@
 # Think of some good inputs to test this functionality, write down at least three
 # examples and verify that the output for these examples is correct.
 
+
+def sorting(*starting_list):
```
Comment (PixelWelt) [inline Exercises/LinusLausberg/exercies_02:80]: nice use of args

File: Exercises/LinusLausberg/exercies_02
Comment (PixelWelt) [inline Exercises/LinusLausberg/exercies_02:1]: please add the .py extension to your file

**Review by atrejut — CHANGES_REQUESTED** [review]
please leave the template in place :)

**Review by atrejut — APPROVED** [review]
thanks, changing only the correct file now :)

**Pull request merged**